### PR TITLE
feat 1358: statuses on roles

### PR DESCRIPTION
**Pull request by stephanieboyd**

### CODE REVIEW

File: gov_users/views/roles_views.py [hunk 1]
```diff
@@ -36,9 +36,7 @@ def get(self, request):
 
     @swagger_auto_schema(request_body=RoleSerializer, responses={400: "JSON parse error"})
     def post(self, request):
-        """
-        Create a role
-        """
+        """ Create a role """
```
Comment (BenAndrew-Kainos) [inline gov_users/views/roles_views.py]: 👀

File: static/statuses/views.py [hunk 2]
```diff
@@ -2,6 +2,7 @@
 from rest_framework.views import APIView
 from rest_framework.status import HTTP_200_OK
 
+from conf.authentication import GovAuthentication
```
Comment (BenAndrew-Kainos) [inline static/statuses/views.py]: Unused import?

File: gov_users/serializers.py [hunk 3]
```diff
@@ -26,10 +27,11 @@ class RoleSerializer(serializers.ModelSerializer):
     organisation = PrimaryKeyRelatedField(queryset=Organisation.objects.all(), required=False, allow_null=True)
     type = serializers.ChoiceField(choices=UserType.choices)
     name = serializers.CharField(max_length=30, error_messages={"blank": strings.Roles.BLANK_NAME},)
+    statuses = PrimaryKeyRelatedField(queryset=CaseStatus.objects.all(), many=True, required=False)
 
     class Meta:
         model = Role
-        fields = ("id", "name", "permissions", "type", "organisation")
+        fields = ("id", "name", "permissions", "type", "organisation", "statuses")
```
Comment (andytaylorkainos) [inline gov_users/serializers.py]: add the comma at the end of the tuple pls

Comment (stephanieboyd) [reply gov_users/serializers.py]: Added

File: static/statuses/views.py (hunk 2 above)
Comment (stephanieboyd) [reply static/statuses/views.py]: Removed

File: static/statuses/views.py [hunk 4]
```diff
@@ -1,7 +1,6 @@
 from django.http import JsonResponse
 from rest_framework.views import APIView
 from rest_framework.status import HTTP_200_OK
-
```
Comment (BenAndrew-Kainos) [inline static/statuses/views.py]: Typo?

**Pull request merged**